doc/release/new-policy.rst

```diff
@@ -0,0 +1,159 @@
+Release Policy Project
+======================
+
+Summary
+-------
+
+Tarantool release policy changes to become more clear and intuitive.
+The new policy uses a `SemVer-like <https://semver.org/>`__ versioning format,
+and introduces a new version lifecycle with more long-time support series.
+This document explains the new release policy, versioning rules, and release series lifecycle.
+
+The new release policy replaces :doc:`the legacy policy <release/policy>`
+for the versions in 2.x.y series since 2.TBD.0,
+and for the future major series ones (3.0.0+) as well.
+
+The topics below describe the new versioning policy in more detail.
+
+Versioning
+----------
+
+Production-ready (release) version labels consist of major, minor and patch numbers:
+
+..  code-block:: text
+
+    MAJOR.MINOR.PATCH
+
+Versions in development and release candidates use the same pattern with an additional suffix:
+
+..  code-block:: text
+
+    MAJOR.MINOR.PATCH-<pre-release suffix>
+
+There are four stages of development before a release:
+
+#.  Alpha (``MAJOR.MINOR.PATCH-alphaN``)
+#.  Beta (``MAJOR.MINOR.PATCH-betaN``)
+#.  Release candidate (``MAJOR.MINOR.PATCH-rcN``)
+#.  Nightly build (``MAJOR.MINOR.PATCH-dev``)
+
+A release series goes through a set of alpha, beta and release candidate versions,
+and eventually gets released.
```

Comment:
```suggestion
A release series is a succession of alpha, beta, and release candidate versions,
which is eventually concluded by the release version.
```
Currently it reads as though the "series" - the multitude of versions - eventually gets released, while we are speaking about a single version (the final release) only.